abrute

```diff
@@ -0,0 +1,74 @@
+.\" Manpage for abrute
+.\" Contact alcamech@gmail.com to correct errors or typos.
+.TH man 1 "12 October 2017" "1.0" "abrute man page"
+.SH NAME
+abrute \- Multi-threaded AES Brute Force File Decryption
+.SH SYNOPSIS
+abrute <RANGE> <CHARACTERS> [OPTIONS] -- <TARGET>
+.SH DESCRIPTION
+Multi-threaded AES Brute Force File Decryption
+.SH OPTIONS
+<RANGE>         Single digit or a range 4:6 for password length.
+.PP
+<CHARACTERS>    Characters to use in password attempt. Don't use quotes
+                unless they may be in the password. Backslash may escape
+                characters such as space.
+.PP
+-a, --adjacent  Set a limit for allowed adjacent characters. Zero will
+                not allow any characters of the same kind to neighbor
+                in the attempts.
+.PP
+-s, --start     Starting character sequence to begin with.
+.PP
+-z, --zip       Use `unzip` decryption instead of `aescrypt`.
+.PP
+<TARGET>        Target file to decrypt.  The target must be preceeded
+                by a double dash: -- target.aes
+.PP
+-h, --help      Prints help information.
+.PP
+-v, --version   Prints version information.
+.SH EXAMPLES
+Decrypts a file with a range of 4, characters of 1234, and a target path to the
+example.file.aes
+
+abrute 4 1234 -- test/data/example.file.aes 1>/dev/null
+
+Decrypts a file with a range of 4, characters of 1234, an option of beginning
+with a character sequence of 2222, and a target path to the example.file.aes
+
+abrute 4 1234 -s 2222 -- test/data/example.file.aes 1>/dev/null
+
+Decrypts a file with a range of 4, characters of 1234, an option of setting the
+allowed adjacent characters to be 0, and a target path ot the example.file.aes
+
+abrute 4 1234 -a 0 -- test/data/example.file.aes 1>/dev/null
+.SH COPYRIGHT
+Copyright 2017 Daniel P. Clark
+
+Permission is hereby granted, free of charge, to any person obtaining
+a copy of this software and associated documentation files (the
+"Software"), to deal in the Software without restriction, including
+without limitation the rights to use, copy, modify, merge, publish,
+distribute, sublicense, and/or sell copies of the Software, and to
+permit persons to whom the Software is furnished to do so, subject to
+the following conditions:
+
+The above copyright notice and this permission notice shall be
+included in all copies or substantial portions of the Software.
+
+THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND,
+EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND
+NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE
+LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION
+WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+.SH REPORTING BUGS
+Report bugs to <https://github.com/danielpclark/abrute/issues>
+.SH BUGS
```

Comment:
At this time we don't need a BUGS section.

Reply:
done